core/server/src/main/java/alluxio/master/MasterContext.java

```diff
@@ -11,58 +11,31 @@
 
 package alluxio.master;
 
-import alluxio.Configuration;
-
 import javax.annotation.concurrent.ThreadSafe;
 
 /**
- * A MasterContext object stores {@link Configuration}.
+ * A singleton for storing shared master state.
  */
 @ThreadSafe
 public final class MasterContext {
   private MasterContext() {} // to prevent initialization
 
   /**
-   * The static configuration object. There is only one {@link Configuration} object shared within
-   * the same master process.
-   */
-  private static Configuration sConfiguration = Configuration.createServerConf();
-
-  /**
    * The {@link MasterSource} for collecting master metrics.
    */
   private static MasterSource sMasterSource = new MasterSource();
 
   /**
-   * @return the {@link Configuration} for the master process
-   */
-  public static Configuration getConf() {
-    return sConfiguration;
-  }
-
-  /**
    * @return the {@link MasterSource} for the master process
    */
   public static MasterSource getMasterSource() {
     return sMasterSource;
   }
 
   /**
-   * Resets the master context, for test only.
-   * TODO(binfan): consider a better way to mock test configuration
+   * Resets the master context.
    */
   public static void reset() {
```

Comment:
I think we can drop this reset method now, or at least use it very rarely (I don't think tests care about the master source).